meinberlin/apps/maps/assets/map_choose_polygon_with_preset.js

```diff
@@ -0,0 +1,247 @@
+/* global django */
+
+const $ = require('jquery')
+// const L = require('leaflet')
```

Comment:
this is an optimisation to only load Leaflet once. but i am not sure if i like it. 
what is your opinion? packaging leaflet to the map_with_preset bundle or using leaflet from the `window.L` scope while ensuring from the python side that is is loaded (`maps/widgets.py`)